src/app/components/FigmaLoading.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { styled } from '@/stitches.config';
+import { useDispatch } from 'react-redux';
+import Icon from './Icon';
+import { postToFigma } from '@/plugin/notifiers';
+import { MessageToPluginTypes } from '@/types/messages';
+import { Tabs } from '@/constants/Tabs';
+import * as pjs from '../../../package.json';
+import Button from './Button';
+import Stack from './Stack';
+import { Dispatch } from '../store';
+
+const FgLoadingScreen = styled('div', {
+  display: 'grid',
+  background: 'Black',
+  padding: '$8',
+  height: 'inherit',
+  color: 'White',
+  alignItems: 'center',
+  justifyContent: 'center',
+  overflowX: 'hidden',
+  overflowY: 'scroll',
+  '& > div > button:hover': {
+    color: 'wheat',
+    background: 'black !important'
+  },
+  '& > div > button': {
+    textDecoration: 'underline',
+    width: 'inherit'
+  },
+  '.rotate > svg': {
+    filter: 'invert(99%) sepia(4%) saturate(2476%) hue-rotate(86deg) brightness(118%) contrast(119%)'
```

Comment:
You can color the spinner by just specifying
```suggestion
    color: '$fgLoadingScreen'
```

Reply:
The problem is that the **color** attribute doesn't seem to work to change the spinner color. Now default loading spinner is defined as black in the `IconLoading.txs` SVG file. If you don't like filter attr to change color, I think we need to create another SVG to define color as white.

Reply:
Lets do this then: change the spinner color on the svg from black to 'currentColor' as in all the other svg's. Then it will be able to pick up the color attribute, no need for a second svg then.

Reply:
done. I removed initial-loading.svg.